routes/v1.js

```diff
@@ -58,9 +60,11 @@ function addRoutes(app, peliasConfig) {
     search: createRouter([
       sanitisers.search.middleware,
       middleware.types,
+      middleware.calcSize(),
```

Comment:
only needed for `/search` and `/reverse` as all other endpoints don't support `size` changes.